WcaOnRails/config/locales/de.yml

```diff
@@ -2724,6 +2792,18 @@ de:
       register_link_html: 'Melde dich %{here} für diesen Wettbewerb an.'
       #original_hash: 45834fd
       register_below_html: Melde dich unten für diesen Wettbewerb an.
+      #original_hash: c5dfcc7
+      part_of_a_series: Dieser Wettbewerb ist Teil einer Wettbewerbsserie.
+      #original_hash: 171e593
+      part_of_a_series_list: >-
+        Dieser Wettbewerb ist Teil der %{name} Wettbewerbsserie. Andere
+        Wettbewerbe in dieser Serie sind:
+      #original_hash: 09fa333
+      series_registration_warning_html: >-
+        Beachte: Du kannst dich für alle Wettbewerbe einer Serie anmelden, aber
+        nur eine dieser Anmeldungen wird bestätigt und akzeptiert. <b>Du wirst
+        die Erlaubnis zur Teilnahme nur für eine der Wettbewerbe der Serie
```

Comment:
Typo: "für eine**n** der Wettbewerbe"

Reply:
Right, noticed this as well now after looking at it in the diff :) thanks!